sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1541,6 +1541,19 @@ def repeat(self, repeats, axis):
           "repeat(repeats=) value must be an int or a "
           f"DeferredSeries (encountered {type(repeats)}).")
 
+  @frame_base.with_docs_from(pd.Series)
```

Comment:
The docs seem sparse on this one, can you add a manual test? (Granted, the implementation is trivial with now, but if this TODO is addressed it'd be nice if existing tests caught anything that went wrong.)

Reply:
Hm good thing you asked for this. When I wrote a test for this I realized this is actually order-sensitive. It returns indexes that can be used with loc to impose the sorted order, so the result depends on the order of the data that is observed by argsort.

I think I had in mind that what was returned was "this element is the Nth largest" which would be independent of the input ordering. I think we should just make this WontImplement(order-sensitive). The rest of this PR could be useful though.

Reply:
Yeah, WontImplement(order-sensitive) seems the right thing here.

Reply:
Done. Also updated the logic for indexing loc with a DeferredSeries of labels to be more general (not just the integer dtype case), and made loc available on `DeferredSeries`. Could you take another look?